test/index.test.ts

```diff
@@ -223,7 +208,7 @@ test('allows transforming request options', async (test) => {
 
   const axiosFetch = buildAxiosFetch(client, transformer);
 
-  const init: FetchInit = { extra: 'options' };
+  const init: RequestInit = { method: 'POST' };
```

Comment:
Is this a change in API? It looks like this test is supposed to use the `options` method

Reply:
Hmmm yeah, you are right. Sorry, I forgot to mention that in the PR description, I should have mentioned it.

As already mentioned earlier I have used proper `fetch` spec-compliant types definition from `node-fetch` in the definition of [`AxiosFetch` here](https://github.com/lifeomic/axios-fetch/pull/86/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R7). With that I have removed your custom interface `FetchInit` that was adding this extra `extra` property, so yes this is a change in API.

If this is important for you, I could bring back the `FetchInit` by extending the `RequestInit` interface with `extra` property and keep this part of test as well.

Reply:
I believe that the support for extra options is desired.. If we took the time to add a test for it, I would strongly prefer that support be left in to avoid breaking a consumer who relies on it.